Source/BusinessMomentsAdapters/Backend/public/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
```

Comment:
Is this file needed? @einari

Reply:
To be honest, I can't remember why its there and if it is needed :) 
When we containerize it - the backend serves the static content through an Express middleware and it is set to look for things in the public folder. 

This could be the result of an overprotective strategy of letting the dev know if they point their browser to the backend during development that they're looking in the wrong place instead of getting a 404 and spending time trying to figure out what is giving this.